cmake/quickjs.cmake

```diff
@@ -29,7 +29,7 @@ if("sgx" IN_LIST COMPILE_TARGETS)
   )
   target_compile_options(
     quickjs.enclave PUBLIC -nostdinc -DCONFIG_VERSION="${QUICKJS_VERSION}"
-                           -DEMSCRIPTEN
+                           -DEMSCRIPTEN -DENABLE_STACK_CHECK
```

Comment:
define CONFIG_STACK_CHECK and avoid having to patch quickjs?

Reply:
Genius.